usage-tests/cra/src/App.tsx

```diff
@@ -0,0 +1,58 @@
+import { howMuch } from "howmuchcarbon";
+import { useMemo, useState } from "react";
+import styled from "styled-components";
+import "./App.css";
+
+const App = () => {
+  const [queryString, setQueryString] = useState("");
+  const results = useMemo(() => howMuch(queryString), [queryString]);
```

Comment:
This `useMemo` is useless here:

```suggestion
  const results = howMuch(queryString);
```